lib/fpclassify.c

```diff
@@ -0,0 +1,48 @@
+/* Replacement function for isnan/isinf/isfinite for processors using
+ * IEEE 754 floating point format.
+ * Copyright (C) 2017, bel2125
```

Comment:
This caught me off guard, as I've not received a contribution which includes its own copyright. However, looking up what this implies it does not really change anything. Because you are not transferring the copyright to another legal entity or person you retain the copyright of your committed changes regardless of adding a header.

You are welcome to include the copyright header, but I'm not sure that it is needed, as the copyright still stays with you. Do you see copyright headers from individual contributors in other open source projects?